user-impl/src/main/java/com/example/auction/user/impl/UserServiceImpl.java

```diff
@@ -42,6 +43,16 @@ public UserServiceImpl(PersistentEntityRegistry registry, UserRepository userRep
     }
 
     @Override
+    public ServiceCall<NotUsed, Done> logout(UUID userId) {
+        return req -> {
+            return entityRef(userId).ask(PUserCommand.GetPUser.INSTANCE)
+                .thenApply(maybeCredential-> {
+                    return Done.getInstance();
+                });
+        };
+    }
```

Comment:
I don't think it's necessary to have a `logout` call in the `UserService`... this method doesn't really do anything. It can be handled entirely in the web gateway.